src/ImageCarouselReact/widget/components/CarouselItem.tsx

```diff
@@ -10,12 +10,8 @@ import * as React from "ImageCarouselReact/lib/react";
 import ReactDOM = require("ImageCarouselReact/lib/react-dom");
 
 import { IObject } from "../utils/bootstrapUtils";
-// import TransitionEvents from "../utils/TransitionEvents";
 import { Direction } from "./Carousel";
 
-// TODO: This should use a timeout instead of TransitionEvents, or else just
-// not wait until transition end to trigger continuing animations.
-
 export interface ICarouselItemProps extends React.Props<CarouselItem> {
```

Comment:
This interface has semi column the other don't...
Think it can go, just like class definition does not have one either.